src/main.ts

```diff
@@ -21,14 +21,14 @@ export const main = async () => {
       const date = cursor.format("YYYY-MM-DD");
       console.log(`i: ${i}; date: ${date}`);
       const zuoraDataFiles = await cycleDataFilesFromZuora(Stage, zuoraBearerToken, date);
-      const subscriptions1a = subscriptionsDataFileToSubscriptions(zuoraDataFiles.subscriptionsFile);
-      const subscriptions1b = retainCorrectSubscriptions(subscriptions1a)
+      const subscriptions1 = subscriptionsDataFileToSubscriptions(zuoraDataFiles.subscriptionsFile);
+      const subscriptions2 = retainCorrectSubscriptions(subscriptions1)
       const holidaySubscriptionNames = holidayNamesDataFileToNames(zuoraDataFiles.holidayNamesFile);
       console.log(holidaySubscriptionNames);
-      const subscriptions2 = excludeHolidaySubscriptions(subscriptions1b, holidaySubscriptionNames);
+      const subscriptions3 = excludeHolidaySubscriptions(subscriptions2, holidaySubscriptionNames);
       const sentDate = now.format("DD/MM/YYYY");
       const deliveryDate = cursor.format("DD/MM/YYYY");
-      const fileRecords = subscriptionsToFileRecords(subscriptions2, sentDate, deliveryDate);
+      const fileRecords = subscriptionsToFileRecords(subscriptions3, sentDate, deliveryDate);
```

Comment:
I don't think the numbers have any meaning, can we distinguish them somehow?
how about something along these lines
1. currentSubs
2. subsWithoutInvalid
3. subsWithoutHolidayStops

Reply:
Thanks. I have adopted those names 🎉